ansible/vars/platform_configs/upp-t-delivery-eu.yaml

```diff
@@ -14,7 +14,7 @@ worker_pools:
   - {id: 2, instance_type: m4.large, count: 3, role: mongo-worker, subnets: 3, dedicatedtaint: mongo}
   - {id: 3, instance_type: c4.xlarge, count: 1, role: kafka-worker, subnets: 1, dedicatedtaint: kafka}
   - {id: 4, instance_type: r4.2xlarge, count: 3, role: neo4j-worker, subnets: 3, dedicatedtaint: neo4j}
-  - {id: 5, instance_type: r4.2xlarge, count: 1, role: neo4j-read-replica, subnets: 3, dedicatedtaint: neo4j-read-replica}
+  - {id: 5, instance_type: r4.2xlarge, count: 1, role: neo4j-read-replica, subnets: 1, dedicatedtaint: neo4j-read-replica}
```

Comment:
We need to limit it, after all, as we need a persistent volume as EBS that is in zone A.